src/context/AppProvider/AppContext.tsx

```diff
@@ -62,6 +63,8 @@ export const AppProvider = ({ children }: { children: React.ReactNode }) => {
   const { isSnapInstalled } = useIsSnapInstalled()
   const { close: closeModal, open: openModal } = useModal('ledgerOpenApp')
 
+  // Previously <TransactionsProvider />
```

Comment:
Not an AI comment 😄  that's actually here for the purpose of grepping `TransactionsProvider` - something something muscle memory